### PR TITLE
GQL Playground: Only run in dev mode

### PR DESCRIPTION
**Pull request by Tobbe**
Fixes https://github.com/redwoodjs/redwood/issues/1591

### CODE REVIEW

File: packages/api/src/functions/graphql.ts [hunk 1]
```diff
@@ -95,7 +95,7 @@ export const createGraphQLHandler = ({
   onHealthCheck,
   ...options
 }: GraphQLHandlerOptions = {}) => {
-  const isDevEnv = process.env.NODE_ENV !== 'production'
+  const isDevEnv = process.env.NODE_ENV === 'development'
```
Comment (Tobbe) [inline packages/api/src/functions/graphql.ts:98]: I thought about adding `|| process.env.NODE_ENV === 'test'` as well, but the playground isn't needed when running tests, right? And neither is introspection I guess. Maybe we want debug. What do you all think?

Comment (peterp) [reply packages/api/src/functions/graphql.ts:98]: Agreed!

**Review by peterp — APPROVED** [review]
🔥 nice job Tobbe!

**Comment by dac09** [comment]
This is great! 🎉thanks @Tobbe

**Review by dac09 — APPROVED** [review]

**Pull request merged**